closure/compiler/closure_js_library.bzl

```diff
@@ -45,7 +45,8 @@ def create_closure_js_library(
         deps = [],
         exports = [],
         suppress = [],
-        lenient = False):
+        lenient = False,
+        internal_descriptors = []):
```

Comment:
Could we use a more descriptive name than `internal_descriptors`? 
(I know that it is called that in `_closure_js_library_impl`, but this is public and might be harder to change later)